src/main/java/io/codelex/classesandobjects/exercises/Exercise7/Dog.java

```diff
@@ -0,0 +1,48 @@
+package io.codelex.classesandobjects.exercises.Exercise7;
+
+public class Dog {
+    private String name;
+    private String sex;
```

Comment:
laba vieta priekš enum

Reply:
Tajā laikā par enum vēl nezināju. To izdarīšu pē lekcijas.